.vscode/launch.json

```diff
@@ -5,16 +5,22 @@
     "version": "0.2.0",
     "configurations": [
         {
-            "name": "Launch",
+            "name": "Launch a test",
             "type": "go",
             "request": "launch",
             "mode": "auto",
-            "program": "${fileDirname}",
+            "program": "${file}",
+            "args": [
+                "-test.v",
+                "-test.run",
+                "^${selectedText}$"
+            ],
             "env": {
-                "TF_ACC": "1"
+                "TF_ACC": "1",
             },
-            "args": [],
-            "buildFlags": "-v -tags=all"
+            "buildFlags": "-v -tags=all",
```

Comment:
but, I can see the variable contents when I debug? Can you test out it's not working as intended?

Reply:
Well personally I had the issue with variables that contain pointers to structs as I remember correctly. So perhaps with simple variables its not an issue at all.

![image](https://user-images.githubusercontent.com/14177833/87542391-2a4ed600-c6a3-11ea-860d-a681b96763ad.png)

Reply:
Do you still have issues? I was not able to reproduce this:

![image](https://user-images.githubusercontent.com/12761692/87781369-78411680-c828-11ea-9e69-5d363cf82dbf.png)

Reply:
Because I always have the possibility to build a "debug version" of the provider by using the `build[.sh|.ps1]` scripts, or change the `buildFlags` , I don't insist having the `gcflags` inside the VSCode configuration.

Reply:
Setting the flag results in error:
```sh
go test: gcflags flag may be set only once
run "go help test" or "go help testflag" for more information
Process exiting with code: 1
```

Flag is already set

Reply:
AHA! That's why you never had issues, when you start debugging directly through VSCode and do not by attaching to a running process like I did. The guys at VSCode are clever enough to set this flag to ensure you can see all variables and stuff 👍